drake/examples/Quadrotor/Quadrotor.m

```diff
@@ -113,13 +113,13 @@ function visualizePlan(x,lcmgl)
   methods (Static)
     
     function runOpenLoop
-      r = Quadrotor('lidar');
+      r = Quadrotor();
```

Comment:
but i really want to leave the lidar on so it’s getting coverage on the build servers…

> On Sep 28, 2015, at 12:02 PM, John Carter <notifications@github.com> wrote:
> 
> In drake/examples/Quadrotor/Quadrotor.m <https://github.com/RobotLocomotion/drake/pull/1176#discussion_r40569852>:
> 
> > @@ -113,13 +113,13 @@ function visualizePlan(x,lcmgl)
> >    methods (Static)
> >      
> >      function runOpenLoop
> > -      r = Quadrotor('lidar');
> > +      r = Quadrotor();
> I turned this off to stop bullet from complaining. With it on I was getting this:
> 
> In BulletModel::findClosestPointsBtwElements: No closest point found!
> —
> Reply to this email directly or view it on GitHub <https://github.com/RobotLocomotion/drake/pull/1176/files#r40569852>.
>